### ISSUE TITLE
Brief flicker when opening/closing tabs

### ISSUE THREAD

**Issue by billyjanitsch** [MoOx/atom-smart-tab-name#6]
Whenever you open or close a tab, there's a brief flicker across the entire tab bar where each tab is first rendered unprocessed (`<filename>.<ext>`, aligned left), then rendered again correctly formatted (filename magic, center aligned).

Repro: open a file `foo.bar` in a tab, then alternate between `cmd+n` and `cmd+w` to quickly open and close a new tab. Notice the flickering left-aligned `foo.bar`.

**Comment by MoOx** [comment]
Indeed, it's probably the time that this plugin can be applied. Not sure if we can improve this.
